Movies-Selection/Movies-Selection/Helpers/ImageService/LoadImageService.swift

```diff
@@ -0,0 +1,22 @@
+// LoadImageService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+import UIKit
+/// LoadImageService
+protocol ImageServiceProtocol {
+    func getImage(url: URL, completion: @escaping (UIImage) -> ())
+}
+
+final class ImageService: ImageServiceProtocol {
+    // MARK: - Public Methods
+
+    func getImage(url: URL, completion: @escaping (UIImage) -> ()) {
+        let imageAPIService = ImageAPIService()
+        let cacheImageService = CacheImageServices()
+        let proxy = ImageProxy(service: imageAPIService, cacheImageService: cacheImageService)
+        proxy.loadImage(url: url) { image in
+            completion(image)
```

Comment:
```suggestion
            completion(image)
```
а если успешный сценарий все будет ок а о плохом не унает юзер надо обработать Result